exercises/practice/armstrong-numbers/.meta/config.json

```
@@ -14,7 +14,6 @@
     ],
     "editor": [
       "data.csv",
```

```suggestion
      "data.csv"
```
To make configlet CI happy